programs/server/config.xml

```diff
@@ -60,6 +60,18 @@
           </logger>
         </levels>
         -->
+        <!-- Structured log formatting:
+        
+        You can specify log format (for now, JSON only). In that case, the console log will be printed
+        in JSON format.
+        For example, like this below:
+        {"datetime":"2022.03.28 12:17:54.827035","thread_id":"790151","query_id":"2441139","priority":"Debug","source":"KeeperTCPHandler","message":"Received close event with xid 2147483647 for session id #7"}  
+        To enable JSON logging support, just uncomment <format> tag and add <json> tag. 
+        Having the <json> tag will make it work. For better understanding/visibility, you can add "true" or "1".
+        -->
+        <format>
+            <json>true</json>
+        </format>
```

Comment:
Need to write a test.

Reply:
For better consistency, let's make JSON formatting exactly as JSONEachRow formatting of system.text_log.

Reply:
Any idea of what kind of test should it run? My guess is, if the `config` has `json` in it, the test program should grab log message and see if the message is in correct log format or not. Anything else? Also, can you hint any test file to understand how to write a test function for this?

Reply:
I can be integration test in `tests/integration`.
It can check that server writes logs in JSON, that JSON is valid, and not necessarily anything else.